sparse_strips/vello_common/src/tile.rs

```diff
@@ -17,22 +17,22 @@ pub const MAX_LINES_PER_PATH: u32 = 1 << 31;
 /// namely if we have multiple lines crossing the same 4x4 area!
 #[derive(Debug, Clone, Copy, Eq, PartialEq, PartialOrd, Ord)]
 #[repr(C)]
-pub struct Tile {
-    /// The index of the tile in the x direction.
-    pub x: u16,
-
-    /// The index of the tile in the y direction.
-    pub y: u16,
-
-    /// The index of the line this tile belongs to into the line buffer, plus whether the line
-    /// crosses the top edge of the tile, packed together.
+pub struct Tile(
+    /// The packed tile data.
     ///
-    /// The index is the unsigned number in the 31 least significant bits of this value.
+    /// The layout is as follows, with the bit indices from least to most significant.
     ///
-    /// The last bit is 1 if and only if the lines crosses the tile's top edge. Lines making this
-    /// crossing increment or decrement the coarse tile winding, depending on the line direction.
-    pub packed_winding_line_idx: u32,
-}
+    /// [ 0, 31): The 31-bit index of the line this tile belongs to into the line buffer, plus
+    /// whether the line crosses the top edge of the tile, packed together;
```

Comment:
> plus whether the line crosses the top edge of the tile, packed together;

That's not quite right, is it? [0, 31) is just the pure index, no other info packed into it.

Reply:
Indeed, that was a vestigial comment from this PR's second commit.